frontend/apps/core-authnz-frontend/package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "admin",
+  "name": "@nrc.no/core-authnz-frontend",
```

Comment:
I've been meaning to ask, is this the scope we want to be using? I'll reserve the npm org if so.